.gitignore

```diff
@@ -54,6 +54,7 @@ perf.data*
 
 # insta snapshot temp file:
 *.pending-snap
+*.snap.new
```

Comment:
Can we leave out that line please? 
```suggestion
```

It's quite useful for me to see info about snapshot related failures in Git tools.